kubernetes/zeus/zeus/worker-deployment.yaml

```diff
@@ -1,10 +1,23 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: worker
+  namespace: default
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: worker
+  minReplicas: 1
+  maxReplicas: 6
+  targetCPUUtilizationPercentage: 60
```

Comment:
You can tweak this to whatever you want, but this just says if > 60%, adds more workers until it settles at less than 60%. Up to the maxReplicas value.